script/enterprise-migrations

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+PATH="~/bin:$PATH"
+eval "$(perl -I ~/perl5/lib/perl5/ '-Mlocal::lib')"
+
+# Depends on PGHOST, PGPASSWORD, PGDATABASE, etc
+createdb
+
+# Use the LOGS_DATABASE_URL we use elsewhere but make it fit the DATABASE_URI format
+# sqitch expects
+sqitch --plan-file db/sqitch-enterprise.plan deploy db:$LOGS_DATABASE_URL
+sqitch --plan-file db/sqitch-enterprise.plan verify db:$LOGS_DATABASE_URL
```

Comment:
I'm sorry to delay this merge further, but I don't like the separate plan file approach.  I would rather use a single plan file and explicitly deploy with `--log-only` in order to skip over the partman-related migrations.  I'll go ahead and commit to this branch rather than try to describe what I mean in a comment 😁